sdk/core/System.ClientModel/api/System.ClientModel.net6.0.cs

```diff
@@ -107,6 +107,10 @@ public partial class HttpClientPipelineTransport : System.ClientModel.Primitives
         protected sealed override void ProcessCore(System.ClientModel.Primitives.PipelineMessage message) { }
         protected sealed override System.Threading.Tasks.ValueTask ProcessCoreAsync(System.ClientModel.Primitives.PipelineMessage message) { throw null; }
     }
+    public partial interface IJsonModel
+    {
+        System.Collections.Generic.IDictionary<string, object> AdditionalProperties { get; }
```

Comment:
That is a good point, and this is currently an open question under discussion - whether we should merge this feature with the existing AdditionalProperties feature.  It is currently unclear whether we would want multiple property dictionaries on a type -- it sounds like we would prefer a single dictionary that supported both/all three cases (with _serializedAdditionalRawData as well), but the details of how this would work have not been finalized.